website/addons/owncloud/serializer.py

```diff
@@ -0,0 +1,81 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+from website.addons.owncloud.settings import DEFAULT_HOSTS
+from website.util import api_url_for, web_url_for
+from website.addons.owncloud.utils import ExternalAccountConverter
+
+class OwnCloudSerializer(OAuthAddonSerializer):
+
+    addon_short_name = 'owncloud'
+
+    REQUIRED_URLS = []
+
+    # Include host information with more informative labels / formatting
+    def serialize_account(self, external_account):
+        ret = super(OwnCloudSerializer, self).serialize_account(external_account)
+        converted = ExternalAccountConverter(external_account)
```

Comment:
It seems unnecessary to go through the password decryption process here. Why not just use `external_account.oauth_key` below?